test/infrastructure/api/ProfileApiRepository.test.ts

```
@@ -0,0 +1,76 @@
+  }
+});
+
+test("API URLが設定されていない場合、エラーを返す", async () => {
+  // 環境変数をモック
+  const originalEnv = process.env;
+  process.env = { ...originalEnv };
+  delete process.env.NEXT_DRUPAL_API;
+
+  const repo = new ProfileApiRepository();
+  const result = await repo.fetch();
+
```

Biome (1.9.4)

[error] Avoid the delete operator which can impact performance.

Unsafe fix: Use an undefined assignment instead.


(lint/performance/noDelete)